acceptance/nodes/redhat7-64m-64a.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Yes, I was thinking the same thing once I noticed we have existing node configurations.  I'll update the PR with the correct location.